instrumentation/opentelemetry-instrumentation-pyramid/src/opentelemetry/instrumentation/pyramid/callbacks.py

```diff
@@ -104,6 +104,8 @@ def _before_traversal(event):
             ] = request.matched_route.pattern
         for key, value in attributes.items():
```

Comment:
Could you add an example of this usage under `Configuration` in `__init__.py`?

Reply:
Added docstring.